ProjectSettings/VersionControlSettings.asset

```diff
@@ -3,6 +3,6 @@
 --- !u!890905787 &1
 VersionControlSettings:
   m_ObjectHideFlags: 0
-  m_Mode: Visible Meta Files
+  m_Mode: Perforce
```

Comment:
I'm not sure why this got changed. This change should be reverted.
- [ ] Revert